docs/docs-sla.md

```diff
@@ -0,0 +1,10 @@
+---
+id: docs-sla
+title: Documentation SLA
+---
+
+This documentation set and all of its components, such as page urls, content, and design, is subject to change at any time.
```

Comment:
I think this is very ambiguously worded. What is a "documentation set"? I would try rephrase this in a less "legal" way ie:

We are constantly working to improve and expand our documentation. There is no guarantee that links will remain active after the documentation is revised. ....

Reply:
I revamped the tone and context of all this content - LMK what you think.